finished frontend for user service

samrkcs opened the pull request:
Added signup and login pages, cleaned up folder structure

samrkcs committed:
finished frontend for user service

Added signup and login pages, cleaned up folder structure

samrkcs committed:
updated UI

Added navbar, login/logout button and updated UI

Danpythonman commented on tracked-frontend/src/components/layout/Navbar.js:
```diff
@@ -1,13 +1,52 @@
+
+const Navbar = () => {
+  const navigate = useNavigate();
+  const [isLoggedIn, setIsLoggedIn] = useState(false); // Temporary state for auth logic
```
before going further with using the `useState` hook for login state, what are your thoughts on using the `useContext` hook to have this state available to all components that need it?

samrkcs replied:
Yeah, I think that is the plan. It was just written temporarily to test the login/logout buttons for the UI

mahfoozm commented:
Could you add the appropriate routes to interact with the user services based on the docs in #4? You can hit localhost:8081.

samrkcs committed:
Auth and routes update

Changed useState hooks to useContext, Fixed API routes, minor repository structure and UI changes
